gateway-service-baseline-lib/src/main/java/org/hypertrace/gateway/service/baseline/lib/BaselineCalculator.java

```diff
@@ -18,7 +18,7 @@ public static Baseline getBaseline(List<Value> metricValues) {
     return getBaseline(values);
   }
 
-  private static Baseline getBaseline(double[] metricValueArray) {
+  public static Baseline getBaseline(double[] metricValueArray) {
```

Comment:
I just saw that hypertrace-alert-engine is already using gateway-service-api. In that case, we can keep it private.